sentences/en/light_HassLightSet.yaml

```
@@ -22,6 +22,7 @@ intents:
           - "[<numeric_value_set>] <area> brightness [to] <brightness>"
           - "[<numeric_value_set>] <area> [to] <brightness> brightness"
           - "[<numeric_value_set>] <area> [to] <brightness>"
+          - "[<numeric_value_set>] [all] [the] lights [in | of] <area> to <brightness> [brightness]"
```

```suggestion
          - "[<numeric_value_set>] [all] [the] lights [<in> | of] <area> to <brightness> [brightness]"
```

Also, could you provide an example where `of` makes sense? I can't think of any atm.

---

I mostly included that based on the existing sentences. An example could be: "set the lights of the bedroom to 50%".

---

I haven't really heard that phrasing before, but it does make sense. If so, I think it would be better to add `of` directly to the `<in>` expansion rule, don't you think?

---

This could make sense. I'm not too familiar with the structure of the rest of the intents, so I'm not sure if there could be any potential negative impact elsewhere. What do you think?

---

It's fine by me. Move the `of` to the `<in>` general expansion rule and remove it from here.

---

Done.